contracts/gateway/GatewayBase.sol

```diff
@@ -60,19 +63,22 @@ contract GatewayBase is Organized {
         uint256 _changedBounty
     );
 
-    bytes32 constant STAKE_TYPEHASH = keccak256(
-        abi.encode(
-            "Stake(uint256 amount,address beneficiary,MessageBus.Message message)"
-        )
-    );
 
-    bytes32 constant REDEEM_TYPEHASH = keccak256(
-        abi.encode(
-            "Redeem(uint256 amount,address beneficiary,MessageBus.Message message)"
-        )
-    );
+    /* Structs */
+
+    struct MessageRegistration {
```

Comment:
Where it is used? 🤔

Reply:
🤦‍♂️ thank you. It is not used anymore. It was there temporary instead of `Message`. Removed ➖